curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63b60cfaca25bb27edd40f62.md

```diff
@@ -14,13 +14,15 @@ Finally, you need to link your JavaScript file to your HTML. Create a `script` e
 You should have a `script` element.
```

Comment:
Wonder if this text should be changed to be more specific. Maybe "You should have opening and closing `script` tags at the end of your `body` element".

Reply:
Hmm, editable region is exactly between `</main>` and `</body>`, not including them. I'm not sure, but in case of changing hint, it would also make sense to modify instructions.